src/pydocstyle/config.py

```diff
@@ -208,7 +208,10 @@ def _get_config(self, node):
         if path in self._cache:
             return self._cache[path]
 
-        config_file = self._get_config_file_in_folder(path)
+        if self._options.config_file:
```

Comment:
@farmersez Won't calling `self._get_config_file_in_folder(self._options.config_path)` disable the ability to specify a custom config filename? `_get_config_file_in_folder` disregards paths to specific files and instead uses `PROJECT_CONFIG_FILES`.